### PR TITLE
Used Ubuntu 18.04 for docs GitHub action.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Review by pope1ni — APPROVED** [review]
Thanks, makes sense.

Are there any plans to update djangoproject.com to 20.04?

**Comment by smithdc1** [comment]
I can't see the docs github action running?

**Comment by felixxm** [comment]
> I can't see the docs github action running?

It's probably caused by a comment. I'll try to put it elsewhere.

**Comment by felixxm** [comment]
> Are there any plans to update djangoproject.com to 20.04?

Yes, we will try to do it by the end of this year.

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ name: Docs
 
 on:
   pull_request:
-    paths:
-      - 'docs/**'
+    # paths:
+    #  - 'docs/**'
```
Comment (felixxm) [inline .github/workflows/docs.yml:6]: I will revert this before merging.

**Comment by felixxm** [comment]
> > I can't see the docs github action running?
> 
> It's probably caused by a comment. I'll try to put it elsewhere.

Ahh, missing docs changes :facepalm:

**Comment by felixxm** [comment]
OK, `docs` passed. Thanks both for checking :+1:

**Pull request merged**